forch/port_state_manager.py

```diff
@@ -138,11 +138,10 @@ def handle_device_placement(self, mac, device_placement, static=False, expired_v
         if (not expired_vlan or not device_behavior or
                 self._get_vlan_from_segment(device_behavior.segment) == expired_vlan):
             self._process_device_placement(mac, device_placement, static=False)
-            if dynamic_behavior:
-                self._process_device_behavior(mac, DeviceBehavior(), static=False)
-        else:
-            LOGGER.info(
-                'Ignoring vlan expiration for device %s with expired vlan %d', mac, expired_vlan)
+            return True
+        LOGGER.info(
```

Comment:
logging message seems slightly misplaced, since at this point it's not "ignoring" it, it's saying "it should be ignored" -- I'd move this same logging message to forchestrator.py and have it as an else clause of the propagate_placement if statement.